it/platform/curl_test.go

```diff
@@ -114,15 +113,12 @@ func (a *OpenSearchTestSuite) TestCurlGet() {
 		assert.NoError(t, err, "failed to get response")
 		assert.NotNil(t, response)
 		var health struct {
-			Name   string `yaml:"cluster_name"`
 			Nodes  string `yaml:"number_of_nodes"`
 			Status string `yaml:"status"`
 		}
 		assert.NoError(t, yaml.Unmarshal(response, &health))
 		assert.True(t, len(health.Status) > 0)
-		assert.EqualValues(t, "yellow", health.Status)
-		assert.EqualValues(t, "test-cluster", health.Name)
-		assert.EqualValues(t, "1", health.Nodes)
+		assert.True(t, len(health.Nodes) > 0)
```

Comment:
We seem to remove other assertions. Is that ok?

Reply:
I removed an assertion which checks for cluster name. I don't think that is useful. Other asserts are verified as True instead of Equals since those values are very docker specific.